torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -458,6 +458,12 @@ class FullyShardedDataParallel(nn.Module):
         Changing the original parameter variable names after construction will
         lead to undefined behavior.
 
+    .. warning::
+        Passing in `sync_module_states=True` flag requires module to be put
```

Comment:
```suggestion
        Passing in ``sync_module_states=True`` flag requires module to be put
```
<img width="741" alt="Screen Shot 2022-05-19 at 12 29 13 AM" src="https://user-images.githubusercontent.com/31054793/169205203-07328d8a-0974-4e76-8cac-18b94bbc7778.png">

Reply:
In the interest of branch cut I'll fix this in a follow up